tests/devices/qutrit_mixed/test_qutrit_mixed_measure.py

```diff
@@ -0,0 +1,507 @@
+        """Check that the sum of terms method is used when Hamiltonian."""
+        H = qml.Hamiltonian([2], [qml.GellMann(0, 1)])
+        state = qml.numpy.zeros((3, 3))
+        assert get_measurement_function(qml.expval(H), state) is sum_of_terms_method
+
+    def test_sum_sum_of_terms(self):
+        """Check that the sum of terms method is used when sum of terms"""
+        S = qml.prod(*(qml.GellMann(i, 1) for i in range(8))) + qml.prod(
+            *(qml.GellMann(i, 2) for i in range(8))
+        )
+        state = qml.numpy.zeros((3, 3))
+        assert get_measurement_function(qml.expval(S), state) is sum_of_terms_method
+
+    def test_probs_compute_probabilities(self):
+        """Check that compute probabilities method is used when probs"""
+        assert get_measurement_function(qml.probs(), state=1) is calculate_probability
+
+    def test_var_compute_variance(self):
+        """Check that the compute variance method is used when variance"""
+        obs = qml.GellMann(0, 1)
+        assert get_measurement_function(qml.var(obs), state=1) is calculate_variance
+
+
+@pytest.mark.parametrize(
+    "obs",
+    [
+        qml.GellMann(2, 2),
+        qml.GellMann(1, 8),
+        qml.GellMann(0, 5),
+        (qml.GellMann(0, 5) @ qml.GellMann(1, 8)),
+    ],
+)
+@pytest.mark.parametrize("ml_framework", ml_frameworks_list)
+class TestApplyObservableEinsum:
+    """Tests that observables are applied correctly for trace method."""
+
+    num_qutrits = 3
+    dims = (3**num_qutrits, 3**num_qutrits)
+
+    def test_apply_observable_einsum(self, obs, three_qutrit_state, ml_framework):
+        """Tests that unbatched observables are applied correctly to a unbatched state."""
+        res = apply_observable_einsum(
+            obs, qml.math.asarray(three_qutrit_state, like=ml_framework), is_state_batched=False
+        )
+        expected = get_expanded_op_mult_state(obs, three_qutrit_state)
+        expected = expected.reshape([3] * (3 * 2))
+
+        assert qml.math.get_interface(res) == ml_framework
+        assert qml.math.allclose(res, expected)
+
+    def test_apply_observable_einsum_batched(self, obs, three_qutrit_batched_state, ml_framework):
+        """Tests that unbatched observables are applied correctly to a batched state."""
+        res = apply_observable_einsum(
+            obs,
+            qml.math.asarray(three_qutrit_batched_state, like=ml_framework),
+            is_state_batched=True,
+        )
+        expected = [
+            get_expanded_op_mult_state(obs, state).reshape([3] * (3 * 2))
+            for state in three_qutrit_batched_state
+        ]
+
+        assert qml.math.get_interface(res) == ml_framework
+        assert qml.math.allclose(res, expected)
+
+
+class TestMeasurements:
+    """Test that measurements on unbatched states work as expected."""
+
+    @pytest.mark.parametrize(
+        "measurement, get_expected",
+        [
+            (qml.state(), lambda x: math.reshape(x, newshape=(9, 9))),
```

Comment:
How come these are being expressed here as function calls, i.e., `qml.state()` instead of just `qml.state`?

Reply:
qml.state() returns a StateMP object, there is similar code in the `test_measure` file in `default.qubit`